### PR TITLE
MGMT-13327: MGMT-13300: Add support with 4.13 dev preview images

### PR DESCRIPTION
**Pull request by eliorerz**
Add OCP 4.13 release images.
Note: Currently, there are no RHCOS images available, so I pointed the RHCOS images to the 4.12 RHCOS images.

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [x] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [x] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [x] Manual (Elaborate on how it was tested)
- [ ] No tests needed

/cc @osherdp @gamli75

### CODE REVIEW

**Comment by eliorerz** [comment]
Waiting for https://github.com/openshift/release/pull/35408 and https://github.com/openshift/assisted-test-infra/pull/1992

**Comment by gamli75** [comment]
once we will have the 4.12 GA version - I guess that we will have also RHCOS images for 4.12 and not 4.12-ec.2 so let's wait with this PR

**Comment by gamli75** [comment]
what about PR for the image-service 4.13 version?

**Comment by osherdp** [comment]
> once we will have the 4.12 GA version - I guess that we will have also RHCOS images for 4.12 and not 4.12-ec.2 so let's wait with this PR

I think it's unrelated. We can upgrade 4.12 independently once GA happens 

But for the image-service I agree. We need it as first stage

**Comment by eliorerz** [comment]
Created one for the image service https://github.com/openshift/assisted-image-service/pull/111

**Comment by osherdp** [comment]
/hold until we minimize ISO images in image-service
https://github.com/openshift/assisted-test-infra/pull/1989

after it lands it's worth retesting to make sure if does limit all jobs for the relevant images

File: data/default_os_images.json
Comment (osherdp) [inline data/default_os_images.json:97]: maybe we don't need those (not yet requirement to test 4.13 with PowerPC / zSystems)

Comment (eliorerz) [reply data/default_os_images.json]: Done

**Review by adriengentil — APPROVED** [review]

**Pull request merged**
